source/scim/faq.rst

```diff
@@ -0,0 +1,4 @@
+FAQ

```

Comment:
A while page for just a line? Isn't it possible to handle that differently?

Reply:
I've put it at the end of setup GLPi, does that look better ?